src/DTO/IdentifierService/IdentifierErrorResponse.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Insly\Identifier\Client\DTO\IdentifierService;
+
+use Insly\Identifier\Client\DTO\IdentifierService\Response\Error;
+
+class IdentifierErrorResponse
+{
+    /**
+     * @param Error[] $errors
```

Comment:
```suggestion
     * @param array<Error> $errors
```

We are thinking about implementing some rules from [here](https://github.com/kubawerlos/php-cs-fixer-custom-fixers) into our codestyle package. Then we'd force to use generic array style in PHPDocs, but I think we can start doing it right now.